doc/sphinx/language/core/records.rst

```diff
@@ -21,7 +21,7 @@ Defining record types
    .. insertprodn record_definition of_type_inst
 
    .. prodn::
-      record_definition ::= {? > } @ident_decl {* @binder } {? : @sort } {? := {? @ident } %{ {*; @record_field } {? ; } %} {? as @ident } }
+      record_definition ::= {? > } @ident_decl {* @binder } {? : @sort } {? := {? @ident } %{ {? {+; @record_field } {? ; } } %} {? as @ident } }
```

Comment:
Not sure what's best. We get 3 levels of boxes here. Maybe it's fine if the documented grammar accepts more than the implemented grammar?